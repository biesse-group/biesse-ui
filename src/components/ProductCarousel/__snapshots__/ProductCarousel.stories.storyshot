// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Slideshow/ProductCarousel Default 1`] = `
.c24 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 500;
  font-size: 14px;
  line-height: 20px;
  color: #FFF;
}

.c24 a {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 500;
  font-size: 14px;
  line-height: 20px;
  color: #FFF;
}

.c18 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: #6D6E70;
  margin: 0;
}

.c18 a {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: #6D6E70;
}

.c19 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: inherit;
  margin: 0;
}

.c19 a {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 24px;
  color: inherit;
}

.c11 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 26px;
  width: 26px;
  -webkit-flex: 0 0 26px;
  -ms-flex: 0 0 26px;
  flex: 0 0 26px;
  color: inherit;
}

.c11 > svg {
  height: 100%;
  width: 100%;
  color: inherit;
}

.c22 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: 20px;
  width: 20px;
  -webkit-flex: 0 0 20px;
  -ms-flex: 0 0 20px;
  flex: 0 0 20px;
  color: #FFF;
}

.c22 > svg {
  height: 100%;
  width: 100%;
  color: white;
}

.c17 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: bold;
  border: 0;
  border-radius: 50px;
  cursor: pointer;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  line-height: 1;
  -webkit-transition: all 0.2s ease-out;
  transition: all 0.2s ease-out;
  text-transform: uppercase;
  outline: none !important;
  white-space: nowrap;
  font-size: 16px;
  padding: 0px 32px;
  height: 40px;
  background-color: transparent;
  color: #6D6E70;
  border: 1px solid #6D6E70;
}

.c17:hover {
  background-color: #6D6E70;
  color: #FFF;
}

.c25 {
  text-transform: uppercase;
}

.c23 {
  margin-right: 9px;
}

.c21 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: bold;
  border: 0;
  border-radius: 50px;
  line-height: 1;
  -webkit-transition: all 0.2s ease-out;
  transition: all 0.2s ease-out;
  text-transform: uppercase;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 0px 14px 0px 10px;
  height: 30px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #E1523D;
}

.c26 {
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: bold;
  border: 0;
  border-radius: 50px;
  line-height: 1;
  -webkit-transition: all 0.2s ease-out;
  transition: all 0.2s ease-out;
  text-transform: uppercase;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 0px 14px 0px 10px;
  height: 30px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #8080FF;
}

.c3 {
  margin-top: 0;
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 700;
  font-size: 45px;
  text-transform: uppercase;
  color: #FFF;
}

.c15 {
  margin-top: 0;
  font-family: "NB International Pro",-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
  font-weight: 700;
  font-size: 40px;
  text-transform: none;
  color: #6D6E70;
}

.c9 {
  color: #6D6E70;
  background-color: transparent;
  outline: none !important;
  cursor: pointer;
  padding: 0;
  width: 50px;
  height: 50px;
  border: 1px solid #6D6E70;
}

.c9:hover {
  background-color: #6D6E70;
  color: #FFF;
  -webkit-transition: all 0.2s ease-out;
  transition: all 0.2s ease-out;
}

.c1 {
  background-color: #6D6E70;
  color: #FFF;
  padding: 42px 90px;
  height: 440px;
  margin-bottom: 124px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-top-right-radius: 40px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  width: 100%;
  height: 100%;
  max-width: 1600px;
}

.c10 {
  position: absolute;
  bottom: 28px;
  left: -50px;
}

.c12 {
  position: absolute;
  bottom: 28px;
  right: -50px;
}

.c14 {
  width: 100%;
  max-width: 600px;
  position: absolute;
}

.c7 {
  position: absolute;
  text-align: center;
  width: 450px;
  height: 450px;
  pointer-events: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c5 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  top: 40px;
}

.c8 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  top: 40px;
}

.c6 {
  color: #FFF;
  font-size: 220px;
  font-weight: 700;
  position: absolute;
  text-align: center;
  width: 800px;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c4 {
  font-size: 50px;
  margin: 0;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c0 {
  position: relative;
  height: 800px;
}

.c20 {
  margin-top: 20px;
}

.c20 > *:not(:last-child) {
  margin-right: 10px;
}

.c16 {
  margin-left: auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c16 > button:not(:last-child) {
  margin-right: 20px;
}

@media screen and (max-width:991px) {

}

@media screen and (max-width:767px) {

}

@media screen and (max-width:991px) {

}

@media screen and (max-width:767px) {

}

@media screen and (max-width:991px) {

}

@media screen and (max-width:767px) {
  .c18 {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {

}

@media screen and (max-width:767px) {
  .c18 a {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {

}

@media screen and (max-width:767px) {
  .c19 {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {

}

@media screen and (max-width:767px) {
  .c19 a {
    font-size: 14px;
  }
}

@media screen and (max-width:991px) {
  .c3 {
    font-size: 40px;
  }
}

@media screen and (max-width:767px) {
  .c3 {
    font-size: 30px;
  }
}

@media screen and (max-width:991px) {

}

@media screen and (max-width:767px) {

}

@media screen and (max-width:767px) {
  .c9 {
    width: 40px;
    height: 40px;
  }
}

@media screen and (max-width:1399px) {
  .c1 {
    height: 300px;
    margin-bottom: 100px;
  }
}

@media screen and (max-width:991px) {
  .c1 {
    height: 440px;
    margin-bottom: 120px;
    padding: 42px 25px;
  }
}

@media screen and (max-width:767px) {
  .c1 {
    height: 330px;
    margin-bottom: 76px;
    padding: 42px 25px;
  }
}

@media screen and (max-width:991px) {
  .c10 {
    left: 0;
    bottom: -10px;
  }
}

@media screen and (max-width:767px) {
  .c10 {
    left: -10px;
  }
}

@media screen and (max-width:991px) {
  .c12 {
    right: 0;
    bottom: -10px;
  }
}

@media screen and (max-width:767px) {
  .c12 {
    right: -10px;
  }
}

@media screen and (max-width:767px) {
  .c14 {
    padding: 0 30px;
  }
}

@media screen and (max-width:1399px) {
  .c7 {
    width: 300px;
    height: 300px;
  }
}

@media screen and (max-width:991px) {
  .c7 {
    width: 450px;
    height: 450px;
    top: -60%;
  }
}

@media screen and (max-width:767px) {
  .c7 {
    width: 280px;
    height: 280px;
    top: unset;
  }
}

@media screen and (max-width:1399px) {
  .c5 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    top: 20px;
  }
}

@media screen and (max-width:991px) {
  .c5 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    top: 170px;
  }
}

@media screen and (max-width:991px) {
  .c5 {
    display: none;
  }
}

@media screen and (max-width:1399px) {
  .c8 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    top: 20px;
  }
}

@media screen and (max-width:991px) {
  .c8 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    top: 170px;
  }
}

@media screen and (min-width:992px) {
  .c8 {
    display: none;
  }
}

@media screen and (max-width:1399px) {
  .c6 {
    font-size: 150px;
    width: 600px;
  }
}

@media screen and (max-width:991px) {
  .c6 {
    font-size: 220px;
    top: -80px;
  }
}

@media screen and (max-width:767px) {
  .c6 {
    font-size: 110px;
    top: -80px;
  }
}

@media screen and (max-width:767px) {
  .c4 {
    font-size: 30px;
    text-align: center;
  }
}

@media screen and (max-width:767px) {
  .c16 {
    -webkit-order: 100;
    -ms-flex-order: 100;
    order: 100;
    margin-left: 0;
    margin-top: 30px;
    -webkit-flex: 0 0 100%;
    -ms-flex: 0 0 100%;
    flex: 0 0 100%;
  }

  .c16 > button {
    -webkit-flex: 1 1 50%;
    -ms-flex: 1 1 50%;
    flex: 1 1 50%;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <div
      className="c2"
    >
      <h2
        className="c3 c4"
        color="light"
      >
        New products
      </h2>
      <div
        className="c5"
      >
        <div
          className="c6"
          style={
            Object {
              "left": "12.5%",
              "opacity": 1,
              "right": "auto",
              "top": "50%",
              "transform": "translateX(-50%) translateY(-50%) scale(0.2) translateZ(0)",
            }
          }
        >
          KK765
        </div>
        <div
          className="c6"
          style={
            Object {
              "left": "50%",
              "opacity": 1,
              "right": "auto",
              "top": "48px",
              "transform": "translateX(-50%) translateY(0px) scale(1) translateZ(0)",
            }
          }
        >
          ES796
        </div>
        <div
          className="c6"
          style={
            Object {
              "left": "auto",
              "opacity": 1,
              "right": "12.5%",
              "top": "50%",
              "transform": "translateX(50%) translateY(-50%) scale(0.2) translateZ(0)",
            }
          }
        >
          AB564
        </div>
      </div>
      <div
        className="c5"
      >
        <div
          className="c7"
          style={
            Object {
              "left": "12.5%",
              "opacity": 1,
              "right": "auto",
              "top": "100%",
              "transform": "translateX(-50%) translateY(-50%) scale(0.5) translateZ(0)",
            }
          }
        >
          <img
            alt="KK765"
            src="https://dummyimage.com/600x400/000/fff"
            style={
              Object {
                "maxHeight": "100%",
                "maxWidth": "100%",
              }
            }
          />
        </div>
        <div
          className="c7"
          style={
            Object {
              "left": "50%",
              "opacity": 1,
              "right": "auto",
              "top": "10%",
              "transform": "translateX(-50%) translateY(0px) scale(1) translateZ(0)",
            }
          }
        >
          <img
            alt="ES796"
            src="assets/product-carousel-1.png"
            style={
              Object {
                "maxHeight": "100%",
                "maxWidth": "100%",
              }
            }
          />
        </div>
        <div
          className="c7"
          style={
            Object {
              "left": "auto",
              "opacity": 1,
              "right": "12.5%",
              "top": "100%",
              "transform": "translateX(50%) translateY(-50%) scale(0.5) translateZ(0)",
            }
          }
        >
          <img
            alt="AB564"
            src="assets/product-carousel-2.png"
            style={
              Object {
                "maxHeight": "100%",
                "maxWidth": "100%",
              }
            }
          />
        </div>
      </div>
      <div
        className="c8"
      >
        <div
          className="c6"
          style={
            Object {
              "opacity": 1,
              "transform": "none",
            }
          }
        >
          ES796
        </div>
      </div>
      <div
        className="c8"
      >
        <div
          className="c7"
          style={
            Object {
              "opacity": 1,
              "transform": "none",
            }
          }
        >
          <img
            alt="ES796"
            src="assets/product-carousel-1.png"
            style={
              Object {
                "maxHeight": "100%",
                "maxWidth": "100%",
              }
            }
          />
        </div>
      </div>
      <button
        aria-label="prev"
        className="c9 c10"
        direction="prev"
        onClick={[Function]}
      >
        <span
          className="c11"
          size="26px"
        >
          <svg
            viewBox="0 0 50 50"
            xmlns="http://www.w3.org/2000/svg"
            xmlnsXlink="http://www.w3.org/1999/xlink"
          >
            <defs>
              <path
                d="M0 0h38.216v48H0z"
              />
            </defs>
            <g
              fill="none"
              fillRule="evenodd"
            >
              <path
                d="M21.717 7.392Z"
                fill="currentColor"
              />
              <g
                transform="matrix(0 1 1 0 1.108 5.892)"
              >
                <path
                  d="M37.777 18.048 20.17.441a1.311 1.311 0 0 0-.112-.101c-.016-.014-.035-.026-.053-.04-.02-.016-.041-.032-.063-.047-.023-.016-.046-.028-.07-.043a1.924 1.924 0 0 0-.055-.033L19.74.142a.77.77 0 0 0-.058-.027c-.025-.01-.05-.02-.075-.028-.02-.006-.041-.015-.063-.021-.026-.009-.052-.014-.078-.021L19.403.03c-.031-.007-.063-.01-.094-.015l-.052-.007a1.34 1.34 0 0 0-.298 0l-.051.007c-.032.004-.063.008-.093.015-.023.004-.044.01-.066.015-.026.007-.052.012-.078.02-.021.007-.042.016-.062.022l-.075.028a.77.77 0 0 0-.06.027l-.073.035c-.019.011-.038.022-.057.035-.023.013-.047.025-.068.04A1.117 1.117 0 0 0 18.21.3l-.053.04c-.04.031-.076.065-.112.1L.44 18.049a1.5 1.5 0 0 0 2.121 2.121L17.608 5.121v41.38a1.5 1.5 0 0 0 3 0V5.12L35.656 20.17a1.494 1.494 0 0 0 2.121 0 1.5 1.5 0 0 0 0-2.121"
                  fill="currentColor"
                />
              </g>
            </g>
          </svg>
        </span>
      </button>
      <button
        aria-label="next"
        className="c9 c12"
        direction="next"
        onClick={[Function]}
      >
        <span
          className="c11"
          size="26px"
        >
          <svg
            viewBox="0 0 50 50"
            xmlns="http://www.w3.org/2000/svg"
            xmlnsXlink="http://www.w3.org/1999/xlink"
          >
            <defs>
              <path
                d="M0 0h38.216v48H0z"
              />
            </defs>
            <g
              fill="none"
              fillRule="evenodd"
            >
              <path
                d="M28.5 7.392Z"
                fill="currentColor"
              />
              <g
                transform="rotate(90 21.608 27.5)"
              >
                <path
                  d="M37.777 18.048 20.17.441a1.311 1.311 0 0 0-.112-.101c-.016-.014-.035-.026-.053-.04-.02-.016-.041-.032-.063-.047-.023-.016-.046-.028-.07-.043a1.924 1.924 0 0 0-.055-.033L19.74.142a.77.77 0 0 0-.058-.027c-.025-.01-.05-.02-.075-.028-.02-.006-.041-.015-.063-.021-.026-.009-.052-.014-.078-.021L19.403.03c-.031-.007-.063-.01-.094-.015l-.052-.007a1.34 1.34 0 0 0-.298 0l-.051.007c-.032.004-.063.008-.093.015-.023.004-.044.01-.066.015-.026.007-.052.012-.078.02-.021.007-.042.016-.062.022l-.075.028a.77.77 0 0 0-.06.027l-.073.035c-.019.011-.038.022-.057.035-.023.013-.047.025-.068.04A1.117 1.117 0 0 0 18.21.3l-.053.04c-.04.031-.076.065-.112.1L.44 18.049a1.5 1.5 0 0 0 2.121 2.121L17.608 5.121v41.38a1.5 1.5 0 0 0 3 0V5.12L35.656 20.17a1.494 1.494 0 0 0 2.121 0 1.5 1.5 0 0 0 0-2.121"
                  fill="currentColor"
                />
              </g>
            </g>
          </svg>
        </span>
      </button>
    </div>
  </div>
  <div
    className="c13"
  >
    <div
      className="c14"
      style={
        Object {
          "opacity": 1,
          "transform": "none",
          "zIndex": 1,
        }
      }
    >
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "flexWrap": "wrap",
          }
        }
      >
        <h3
          className="c15"
          color="primary"
          style={
            Object {
              "margin": 0,
            }
          }
        >
          ES796
        </h3>
        <div
          className="c16"
        >
          <button
            className="c17"
            type="button"
          >
            Dettagli
          </button>
          <button
            className="c17"
            type="button"
          >
            Confronta
          </button>
        </div>
        <p
          className="c18"
          color="primary"
          style={
            Object {
              "flex": "0 0 100%",
              "marginTop": 15,
            }
          }
        >
          Testate a 2 assi
        </p>
        <p
          className="c19"
          style={
            Object {
              "marginTop": 12,
            }
          }
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nis.
        </p>
        <div
          className="c20"
        >
          <div
            className="c21"
            color="stone"
          >
            <span
              className="c22 c23"
              color="white"
              size="xs"
            >
              <svg
                viewBox="0 0 50 50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M22.972 2.66 2.422 17.59a3.45 3.45 0 0 0-1.253 3.856l7.85 24.158a3.449 3.449 0 0 0 3.28 2.384h25.402a3.45 3.45 0 0 0 3.28-2.384l7.85-24.158a3.45 3.45 0 0 0-1.253-3.856L27.028 2.66a3.448 3.448 0 0 0-4.056 0"
                  fill="currentColor"
                  fillRule="evenodd"
                />
              </svg>
            </span>
            <span
              className="c24 c25"
              color="light"
              size="sm"
            >
              stone
            </span>
          </div>
          <div
            className="c26"
            color="composite"
          >
            <span
              className="c22 c23"
              color="white"
              size="xs"
            >
              <svg
                viewBox="0 0 50 50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M48.008 22.605 27.395 1.992a3.387 3.387 0 0 0-4.79 0L1.992 22.605a3.387 3.387 0 0 0 0 4.79l20.613 20.613a3.387 3.387 0 0 0 4.79 0l20.613-20.613a3.387 3.387 0 0 0 0-4.79"
                  fill="currentColor"
                  fillRule="evenodd"
                />
              </svg>
            </span>
            <span
              className="c24 c25"
              color="light"
              size="sm"
            >
              composite
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
