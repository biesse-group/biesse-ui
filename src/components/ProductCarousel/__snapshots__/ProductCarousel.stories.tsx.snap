// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slideshow/ProductCarousel Default smoke-test 1`] = `
<div class="sc-fUnMCh fXXNRk">
  <div class="sc-jEACwC ipyClk">
    <div class="sc-cPiKLX jNQliY">
      <h2 color="light"
          class="sc-dhKdcB gIFflR sc-dAbbOL cQAQGz"
      >
        New products
      </h2>
      <div class="sc-gFqAkR eaNQrX">
        <div class="sc-ikkxIA jlboWC"
             style="top: 50%; left: 12.5%; right: auto; opacity: 1; transform: translateX(-50%) translateY(-50%) scale(0.2) translateZ(0px);"
        >
          KK765
        </div>
        <div class="sc-ikkxIA jlboWC"
             style="top: 48px; left: 50%; right: auto; opacity: 1; transform: translateX(-50%) translateY(0px) scale(1) translateZ(0px);"
        >
          ES796
        </div>
        <div class="sc-ikkxIA jlboWC"
             style="top: 50%; right: 12.5%; left: auto; opacity: 1; transform: translateX(50%) translateY(-50%) scale(0.2) translateZ(0px);"
        >
          AB564
        </div>
      </div>
      <div class="sc-gFqAkR eaNQrX">
        <div class="sc-fPXMVe jMbAPt"
             style="top: 100%; left: 12.5%; right: auto; opacity: 1; transform: translateX(-50%) translateY(-50%) scale(0.5) translateZ(0px);"
        >
          <img src="https://dummyimage.com/600x400/000/fff"
               alt="KK765"
               style="max-width: 100%; max-height: 100%;"
          >
        </div>
        <div class="sc-fPXMVe jMbAPt"
             style="top: 10%; left: 50%; right: auto; opacity: 1; transform: translateX(-50%) translateY(0px) scale(1) translateZ(0px);"
        >
          <img src="assets/product-carousel-1.png"
               alt="ES796"
               style="max-width: 100%; max-height: 100%;"
          >
        </div>
        <div class="sc-fPXMVe jMbAPt"
             style="top: 100%; right: 12.5%; left: auto; opacity: 1; transform: translateX(50%) translateY(-50%) scale(0.5) translateZ(0px);"
        >
          <img src="assets/product-carousel-2.png"
               alt="AB564"
               style="max-width: 100%; max-height: 100%;"
          >
        </div>
      </div>
      <div class="sc-gFqAkR kFoyy">
        <div class="sc-ikkxIA jlboWC"
             style="opacity: 1; transform: none;"
        >
          ES796
        </div>
      </div>
      <div class="sc-gFqAkR kFoyy">
        <div class="sc-fPXMVe jMbAPt"
             style="opacity: 1; transform: none;"
        >
          <img src="assets/product-carousel-1.png"
               alt="ES796"
               style="max-width: 100%; max-height: 100%;"
          >
        </div>
      </div>
      <button aria-label="prev"
              direction="prev"
              class="sc-dLMFU JIMNl sc-eDPEul fvxrxe"
      >
        <span class="sc-aXZVg dhyslS">
          <svg viewbox="0 0 50 50"
               xmlns="http://www.w3.org/2000/svg"
               xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <defs>
              <path d="M0 0h38.216v48H0z">
              </path>
            </defs>
            <g fill="none"
               fill-rule="evenodd"
            >
              <path d="M21.717 7.392Z"
                    fill="currentColor"
              >
              </path>
              <g transform="matrix(0 1 1 0 1.108 5.892)">
                <path d="M37.777 18.048 20.17.441a1.311 1.311 0 0 0-.112-.101c-.016-.014-.035-.026-.053-.04-.02-.016-.041-.032-.063-.047-.023-.016-.046-.028-.07-.043a1.924 1.924 0 0 0-.055-.033L19.74.142a.77.77 0 0 0-.058-.027c-.025-.01-.05-.02-.075-.028-.02-.006-.041-.015-.063-.021-.026-.009-.052-.014-.078-.021L19.403.03c-.031-.007-.063-.01-.094-.015l-.052-.007a1.34 1.34 0 0 0-.298 0l-.051.007c-.032.004-.063.008-.093.015-.023.004-.044.01-.066.015-.026.007-.052.012-.078.02-.021.007-.042.016-.062.022l-.075.028a.77.77 0 0 0-.06.027l-.073.035c-.019.011-.038.022-.057.035-.023.013-.047.025-.068.04A1.117 1.117 0 0 0 18.21.3l-.053.04c-.04.031-.076.065-.112.1L.44 18.049a1.5 1.5 0 0 0 2.121 2.121L17.608 5.121v41.38a1.5 1.5 0 0 0 3 0V5.12L35.656 20.17a1.494 1.494 0 0 0 2.121 0 1.5 1.5 0 0 0 0-2.121"
                      fill="currentColor"
                >
                </path>
              </g>
            </g>
          </svg>
        </span>
      </button>
      <button aria-label="next"
              direction="next"
              class="sc-dLMFU JIMNl sc-eDPEul fXiuqB"
      >
        <span class="sc-aXZVg dhyslS">
          <svg viewbox="0 0 50 50"
               xmlns="http://www.w3.org/2000/svg"
               xmlns:xlink="http://www.w3.org/1999/xlink"
          >
            <defs>
              <path d="M0 0h38.216v48H0z">
              </path>
            </defs>
            <g fill="none"
               fill-rule="evenodd"
            >
              <path d="M28.5 7.392Z"
                    fill="currentColor"
              >
              </path>
              <g transform="rotate(90 21.608 27.5)">
                <path d="M37.777 18.048 20.17.441a1.311 1.311 0 0 0-.112-.101c-.016-.014-.035-.026-.053-.04-.02-.016-.041-.032-.063-.047-.023-.016-.046-.028-.07-.043a1.924 1.924 0 0 0-.055-.033L19.74.142a.77.77 0 0 0-.058-.027c-.025-.01-.05-.02-.075-.028-.02-.006-.041-.015-.063-.021-.026-.009-.052-.014-.078-.021L19.403.03c-.031-.007-.063-.01-.094-.015l-.052-.007a1.34 1.34 0 0 0-.298 0l-.051.007c-.032.004-.063.008-.093.015-.023.004-.044.01-.066.015-.026.007-.052.012-.078.02-.021.007-.042.016-.062.022l-.075.028a.77.77 0 0 0-.06.027l-.073.035c-.019.011-.038.022-.057.035-.023.013-.047.025-.068.04A1.117 1.117 0 0 0 18.21.3l-.053.04c-.04.031-.076.065-.112.1L.44 18.049a1.5 1.5 0 0 0 2.121 2.121L17.608 5.121v41.38a1.5 1.5 0 0 0 3 0V5.12L35.656 20.17a1.494 1.494 0 0 0 2.121 0 1.5 1.5 0 0 0 0-2.121"
                      fill="currentColor"
                >
                </path>
              </g>
            </g>
          </svg>
        </span>
      </button>
    </div>
  </div>
  <div class="sc-feUZmu dHrNBq">
    <div class="sc-eldPxv hmJrDE"
         style="z-index: 1; opacity: 1; transform: none;"
    >
      <div style="display: flex; align-items: center; flex-wrap: wrap;">
        <h3 color="primary"
            class="sc-kpDqfm iMIeaH"
            style="margin: 0px;"
        >
          ES796
        </h3>
        <div class="sc-fHjqPf vLUvB">
          <button type="button"
                  class="sc-fqkvVR iweJCL"
          >
            Dettagli
          </button>
          <button type="button"
                  class="sc-fqkvVR iweJCL"
          >
            Confronta
          </button>
        </div>
        <p color="primary"
           class="sc-iGgWBj gfkgeR"
           style="margin-top: 15px; flex: 0 0 100%;"
        >
          Testate a 2 assi
        </p>
        <p class="sc-iGgWBj rgmkt"
           style="margin-top: 12px;"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nis.
        </p>
        <div class="sc-hzhJZQ eWtXiM">
          <div color="stone"
               class="sc-imWYAI hmAryj"
          >
            <span color="white"
                  class="sc-aXZVg ctqbVW sc-kAyceB fGnwOA"
            >
              <svg viewbox="0 0 50 50"
                   xmlns="http://www.w3.org/2000/svg"
              >
                <path d="M22.972 2.66 2.422 17.59a3.45 3.45 0 0 0-1.253 3.856l7.85 24.158a3.449 3.449 0 0 0 3.28 2.384h25.402a3.45 3.45 0 0 0 3.28-2.384l7.85-24.158a3.45 3.45 0 0 0-1.253-3.856L27.028 2.66a3.448 3.448 0 0 0-4.056 0"
                      fill="currentColor"
                      fill-rule="evenodd"
                >
                </path>
              </svg>
            </span>
            <span color="light"
                  class="sc-dcJsrY bEhOwU sc-gsFSXq eXWFlb"
            >
              stone
            </span>
          </div>
          <div color="composite"
               class="sc-imWYAI hGlJLP"
          >
            <span color="white"
                  class="sc-aXZVg ctqbVW sc-kAyceB fGnwOA"
            >
              <svg viewbox="0 0 50 50"
                   xmlns="http://www.w3.org/2000/svg"
              >
                <path d="M48.008 22.605 27.395 1.992a3.387 3.387 0 0 0-4.79 0L1.992 22.605a3.387 3.387 0 0 0 0 4.79l20.613 20.613a3.387 3.387 0 0 0 4.79 0l20.613-20.613a3.387 3.387 0 0 0 0-4.79"
                      fill="currentColor"
                      fill-rule="evenodd"
                >
                </path>
              </svg>
            </span>
            <span color="light"
                  class="sc-dcJsrY bEhOwU sc-gsFSXq eXWFlb"
            >
              composite
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
